rts/Map/SMF/ROAM/Patch.cpp

```diff
@@ -88,6 +90,7 @@ void CTriNodePool::Resize(size_t poolSize)
 	// live outside the pool, but KISS)
 	assert((poolSize & 1) == 0);
 	assert(poolSize > 0);
+	LOG_L(L_INFO, "[TriNodePool::%s] to %i ",__func__, poolSize);
```

Comment:
`poolSize` is `size_t` (ie. unsigned)
```suggestion
	LOG_L(L_INFO, "[TriNodePool::%s] to %u ",__func__, poolSize);
```

Reply:
not `%u`, use `_STPF_` for printing `size_t`

Reply:
Possibly even `%zu` because `size_t` could be larger than a regular `unsigned` but I don't think `z` is portable.

Reply:
`%z` is not portable, that's why we use `_STPF_`